web/assets/styles/common.scss

```diff
@@ -1,6 +1,7 @@
 html,
 body {
-    background: var(--body-bg);
+    background: #000;
+    background: var(--body-bg, #000);
```

Comment:
Yes it was cssmin, good catch!
I couldn't get gulp-cssmin to understand the options so I replaced it by the (probably better supported) gulp-clean-css.